staging/kos/cmd/attachment-tput-driver/main.go

```diff
@@ -1297,3 +1309,33 @@ func withRetries(thunk func() bool, minWait, maxWait time.Duration, stopCh <-cha
 	}
 	return true
 }
+
+func printCSVOpsTimes(opsTimes map[uint64]threadOpsSchedule, rate float64, distribution, outputDir string) (err error) {
```

Comment:
The format written here produces a lot of columns --- potentially more than a spreadsheet program will load.

Reply:
Fixed in 49677ba .